src/ansys/aedt/core/visualization/post/common.py

```diff
@@ -555,9 +552,16 @@ def _oproject(self):
         return self._app._oproject
 
     @property
-    def modeler(self):
+    def _modeler(self):
```

Comment:
Could you explain why you want to rename this property as `_modeler` ? I think property should be used to mask implementation details and provide a clean interface so I would prefer to stick with `modeler`

Reply:
modeler is an object of the app:
hfss.modeler

having it internal in post has 2 main purposes:
- avoiding to load modeler when the post class is initialized (before it was initialized in the __init__)
- make it hidden. The purpose of this property is to make coding simpler to methods in Post that requires access to the modeler. It's not intended to be used by users

Reply:
I see but it's a bit strange to hide something using a Python property.
Would it be possible to remove the Python property and add `self._modeler = self._app._modeler` in the `__init__` ? I know it would revert your changes in the `__init__` but it would follow the logic of only providing public stuff in Python properties. If required, we can add a mecanism to load the modeler lazily with an extra attribute like `self.__modeler_loaded`. Also, what's the value of not loading the modeler by default ? Does is make the code much faster ?

Reply:
modeler initialization takes time on big problems. people interested in post processing only may not need at all the modeler. Is all about efficiency. BTW you cannot initialize self._app._modeler because also _modeler is not initialized when you launch pyaedt but first time user uses modeler property

Reply:
Gotcha. I'm still not really happy with this `_modeler` property since property should be things accessible to our users. However, I understand your point. If others are fine with that, I won't oppose firmly but... still feeling we shouldn't use a property here. Would it be possible to refactor the code to avoid that behavior ? It seems to be about encapsulating data so we should be able to reach a good trade off :)